parcels/particleset.py

```diff
@@ -158,17 +161,20 @@ def from_line(cls, fieldset, pclass, start, finish, size, depth=None, time=None,
         :param depth: Optional list of initial depth values for particles. Default is 0m
         :param time: Optional start time value for particles. Default is fieldset.U.time[0]
         :param repeatdt: Optional interval (in seconds) on which to repeat the release of the ParticleSet
+        :param lonlatdepth_dtype: Floating precision for lon, lat, depth particle coordinates.
+               It is either np.float32 or np.float64. Default is np.float32 if fieldset.U.interp_method is 'linear'
+               and np.float64 if the interpolation method is 'cgrid_velocity'
```

Comment:
Do we want all those changes?
I thought we could simply set `lonlatdepth_dtype` through the `**kwargs`. 
(but they were missing in from_line)
So both methods are obviously valid, but I don't know what's the most common one. And if the param is not in the argument list, then should it be in the docstring, or only in the docstring of pset constructor? Let's have a look how such parameters usually work)